### PR TITLE
Support k8s 1.29

### PR DESCRIPTION
**Pull request by toshipp**
Fix: #793

### CODE REVIEW

**Review by satoru-takeuchi — APPROVED** [review]

File: internal/client/client_test.go [hunk 1]
```diff
@@ -1524,6 +1525,7 @@ var _ = Describe("client", func() {
 						svc.Name = configmapName
 						svc.Namespace = configmapNamespace
 						svc.Spec = corev1.ServiceSpec{
+							Type:     corev1.ServiceTypeLoadBalancer,
```
Comment (daichimukai) [inline internal/client/client_test.go:1528]: Note: From k8s 1.29, a Service must have LoadBalancer as its spec.type to set status.loadBalancer.

> Setting the status.loadBalancer of a Service whose spec.type is not LoadBalancer was previously allowed, but any update to the metadata or spec would wipe that field. Setting this field is no longer permitted unless spec.type is LoadBalancer.

https://github.com/kubernetes/kubernetes/blob/e1e17973be8a023cda5f2b8c61a94bc32841acb5/CHANGELOG/CHANGELOG-1.29.md

File: internal/controller/logicalvolume_controller_test.go [hunk 2]
```diff
@@ -142,7 +142,7 @@ var _ = Describe("LogicalVolume controller", func() {
 				AccessModes: []corev1.PersistentVolumeAccessMode{
 					corev1.ReadWriteOnce,
 				},
-				Resources: corev1.ResourceRequirements{
+				Resources: corev1.VolumeResourceRequirements{
```
Comment (daichimukai) [inline internal/controller/logicalvolume_controller_test.go:145]: Note:

> Go API: the ResourceRequirements struct was replaced with VolumeResourceRequirements for use with volumes.

https://github.com/kubernetes/kubernetes/blob/e1e17973be8a023cda5f2b8c61a94bc32841acb5/CHANGELOG/CHANGELOG-1.29.md

**Review by daichimukai — APPROVED** [review]

**Comment by HappyKeanu** [comment]
Hi Team, could you please advise if you are planning to release a new tag with these changes soon?

**Comment by toshipp** [comment]
Our release cycle is monthly, I expect the new release to ship in the 1st week of June.

**Pull request merged**